backbone/src/main/java/org/researchstack/backbone/task/factory/MoodSurveyFactory.java

```diff
@@ -23,6 +23,7 @@
 
 public class MoodSurveyFactory {
 
+    public static final String MoodSurveyIdentifier = "Mood Survey";
```

Comment:
Refactored this class into multiple method calls that can be reused by a subclass for creating the various steps.